openmc/material.py

```diff
@@ -422,7 +427,15 @@ def add_element(self, element, percent, percent_type='ao'):
         else:
             element = openmc.Element(element)
 
-        self._elements[element._name] = (element, percent, percent_type)
+        if expand:
+            if percent_type == 'wo':
+                raise NotImplementedError('Expanding natural element based on '
+                                          'weight percent is not yet supported.')
+            for isotope, abundance in element.expand():
+                self._nuclides[isotope.name] = (
```

Comment:
Gotcha, I should have checked that those properties existed first. It's good to add them in here anyway.